.github/workflows/pr_sync_ci.yml

```
@@ -40,10 +39,10 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
 
+      # run 명령어로 schema 만드는 명령어 실행
+      - name: Into Schema
```

`Into Schema` 라는 이름이 추상적인 느낌이 들기도 하네요 `Execute DDL script`는 어떨까요?
```suggestion
      - name: Execute DDL script
```

---

Create schema 좋은 것 같습니다!